zerver/lib/markdown/__init__.py

```diff
@@ -125,27 +125,22 @@ def normal_compile(pattern: str) -> Any:
     )
 
 
-STREAM_LINK_REGEX = r"""
-                     (?<![^\s'"\(,:<])            # Start after whitespace or specified chars
-                     \#\*\*                       # and after hash sign followed by double asterisks
-                         (?P<stream_name>[^\*]+)  # stream name can contain anything
-                     \*\*                         # ends by double asterisks
-                    """
-
-
-@one_time
-def get_compiled_stream_link_regex() -> Pattern:
-    return verbose_compile(STREAM_LINK_REGEX)
-
-
-STREAM_TOPIC_LINK_REGEX = r"""
-                     (?<![^\s'"\(,:<])             # Start after whitespace or specified chars
-                     \#\*\*                        # and after hash sign followed by double asterisks
-                         (?P<stream_name>[^\*>]+)  # stream name can contain anything except >
-                         >                         # > acts as separator
-                         (?P<topic_name>[^\*]+)     # topic name can contain anything
-                     \*\*                          # ends by double asterisks
-                   """
+# Start after whitespace or specified chars
+# and after hash sign followed by double asterisks
+# stream name can contain anything
+# ends by double asterisks
+STREAM_LINK_REGEX = r"(?<![^\s'\"\(,:<])\#\*\*(?P<stream_name>[^\*]+)\*\*"
+
+
+# Start after whitespace or specified chars
+# and after hash sign followed by double asterisks
+# stream name can contain anything except >
+# > acts as separator
+# topic name can contain anything
+# ends by double asterisks
+STREAM_TOPIC_LINK_REGEX = (
+    r"(?<![^\s'\"\(,:<])\#\*\*(?P<stream_name>[^\*>]+)>(?P<topic_name>[^\*]+)\*\*"
+)
```

Comment:
What's the thinking around removing the line-wrapping on these?

Reply:
oops, sorry I might have changed this during several approaches I tried(that required me to compile without using `VERBOSE` flag) and later forgot to change it back. I have fixed it.